### PR TITLE
Fix #5563 - Force fxa webviews to have mobile UA

### LINKED ISSUE #5563: Firefox account page too large for view on iPhone
Cross posted to fxa https://github.com/mozilla/fxa/issues/2795

image

### PR DESCRIPTION
**Pull request by garvankeeley**
Fixes #5563

### CODE REVIEW

File: Shared/UserAgent.swift [hunk 1]
```diff
@@ -41,13 +41,12 @@ open class UserAgent {
     public static func isDesktop(ua: String) -> Bool {
         return ua.lowercased().contains("intel mac")
     }
-
     public static func desktopUserAgent() -> String {
-        return "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15) AppleWebKit/605.1.15 (KHTML, like Gecko)"
+        return "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15) AppleWebKit/605.1.15 (KHTML, like Gecko) FxiOS/\(AppInfo.appVersion) Mobile/15E148 Safari/605.1.15"
```
Comment (dnarcese) [inline Shared/UserAgent.swift]: Should the desktop UA have the `mobile` part?

Comment (garvankeeley) [reply Shared/UserAgent.swift]: crap, no

**Comment by garvankeeley** [comment]
Closing, this doesn't work on iPad

**Comment by garvankeeley** [comment]
Fixed it for iPad

**Pull request merged**